docs/agents/sematext-agent/agent-troubleshooting.md

```diff
@@ -0,0 +1,38 @@
+title: Agent Troubleshooting
+
+If you are experiencing issues with the Sematext Agent, there are some things you can try.
+
+1. Check the Sematext Agent logs and filter by errors. By default they'll be located at `/opt/spm/spm-monitor/logs/st-agent`. 
+   1. If you see errors related to permission denied while accessing log files or system information, [stop the agent and start it](https://sematext.com/docs/agents/sematext-agent/starting-stopping) again with [elevated privileges](https://sematext.com/docs/agents/sematext-agent/permission-requirements/). 
+   2. If you see errors like `"cloud" collector failed to complete the initial run` check out our [tags common schema docs](https://sematext.com/docs/tags/common-schema/#cloud-tags) and verify your IAM roles are setup correctly.
+   
+
+2. Try restarting the agent. Specific instructions on how to do so depending on your environment can be found [here](https://sematext.com/docs/agents/sematext-agent/starting-stopping).
+
+3. Update the agent to the latest version. For more details on how to do that, see our [Agent Upgrade Instructions](https://sematext.com/docs/monitoring/spm-faq/#agent-updating). You can see which agents need to be updated in the Fleet section of Sematext Cloud.
+![Fleet](../../images/agents/fleet-menu.png)
+If there is an update available for a particular agent, you'll see a warning next to the agent's version.
+
+4. Journal is a buffer we use to store failed events. In case of outages or internal communication issues, the Journal files may grow rapidly and produce high system load. If you are experiencing high CPU usage, do the following:
```

Comment:
"events" reference risks people confusing that with our Events.  I *think* by "events" we mean "any data agent sends to SC"?

Reply:
I changed it for "store data when shipping fails"